specs/darwin/unified_log.table

```diff
@@ -16,12 +20,15 @@ schema([
   Column("category", TEXT, "the category of the os_log_t used", additional=True),
   Column("subsystem", TEXT, "the subsystem of the os_log_t used", additional=True),
   Column("level", TEXT, "the severity level of the entry"),
-  Column("max_rows", INTEGER, "the max number of rows returned (defaults to 100)", additional=True, hidden=True)
+  Column("max_rows", INTEGER, "the max number of rows returned (defaults to 100)", additional=True, hidden=True),
+  Column("predicate", TEXT, "predicate to require (see `log help predicates`)", additional=True, hidden=True)
```

Comment:
Yeah it does merge it together, will note that.

> What happens if multiple predicates are specified?

Great question! If multiple predicates are AND-ed, it silently ignores one of them, which doesn't seem great. I think I'll change it to do an AND by refactoring the logic into `addQueryOp` to be consistent (alternative is probably to return an error on multiple predicate ANDs).

If multiple predicates are OR-ed, then the SQL engine will split it up into multiple table calls. This works fine except for the `timestamp > -1` case, because the log pointer gets moved in between -- the other columns have this issue right now too. 

Would be great to have that throw an error or even work correctly by not advancing pointer until all calls are done, but I'm not quite sure how to do that. It's probably a bigger change to how virtual tables work.

Reply:
I'm not totally sure how it should work, but one thing we talked about on the original PR, was that `timestamp > -1` was a fairly simple case. It wasn't clear if or how we should track things for multiple queries. 

Checksum the query context? Allow other negative numbers? 🤷

Reply:
> I think I'll change it to do an AND by refactoring the logic into addQueryOp to be consistent (alternative is probably to return an error on multiple predicate ANDs).

Quick note I'm changing multiple AND predicates to an error because `predicate` has to match in the returned row as well for SQL engine to not filter it out, so users need to do the AND internally when using it for correct behavior.

> I'm not totally sure how it should work, but one thing we talked about on the original PR, was that timestamp > -1 was a fairly simple case. It wasn't clear if or how we should track things for multiple queries.

Using the currently running scheduled query name seems good as it's simple and consistent with how the optimized events tables work. That wouldn't be too hard to add to this PR. But it doesn't help the issue of the SQL engine making multiple independent calls to the virtual table within a single scheduled query due to an OR condition.